app/main/views.py

```diff
@@ -602,16 +604,38 @@ def user_list_page():
     active = eval_request_bool(request.args.get('active', "true"), True)
     nondivision_users = []
     tags = get_all_tags()
-    list_of_users = User.query.filter_by(is_active=active).all()
+    list_of_users = []
+    list_of_users_all  = User.query.filter_by(is_active=active).all()
     for user in list_of_users:
         if user.division is None:
             list_of_users.remove(user)
             nondivision_users.append(user)
+    #   Search results     
+    if request.method == 'GET':
+        entry = request.args.get('search_input', '')
+        print('list\n', list_of_users_all)
```

Comment:
Remove print statements and extra blank lines in this code `if` statement block.